doc/fixing-broken-sites.md

```diff
@@ -48,6 +50,8 @@ We should also ask the service to to adopt the [EFF Do Not Track policy](https:/
 
 ### External domains too complex to surrogate or replace with placeholders
 
-If nothing else seems to fit, adding the affected domain to the "[yellowlist](/doc/yellowlist-criteria.md)" will make Privacy Badger set the domain to "yellow" (cookie-blocked) instead of "red" (blocked) after seeing it track on three or more sites.
+If nothing else seems to fit, adding the affected domain to the "[yellowlist](/doc/yellowlist-criteria.md)" will make Privacy Badger set the domain to "yellow" ("cookie-blocked") instead of "red" (blocked) after seeing it track on three or more sites.
```

Comment:
I didn't, because "blocked" is a pretty clear general-use word (while "cookie-blocked" is a not-necessarily-obvious piece of Privacy Badger terminology).